buildfile.m

```diff
@@ -0,0 +1,30 @@
+function plan = buildfile
+
+import matlab.buildtool.tasks.TestTask
+import matlab.buildtool.tasks.CleanTask
+
+% Create a plan from task functions
+plan = buildplan(localfunctions);
+
+% Add a task to run tests
+plan("test") = TestTask(SourceFiles="code", ...
+    TestResults="test-results/results.html", ...
+    CodeCoverageResults="code-coverage/results.xml");
+
```

Comment:
We should consider adding `plan("check") = CodeIssuesTask;` as well

Reply:
We might want to resist the urge to teach the build tool here since this repo really focuses on CI configuration. I could even imagine a super simple build file that just replicates the behavior of the `run-tests` and `run-command` steps we show.

```matlab
function plan = buildfile
addpath("code");
plan = buildplan();
plan("test") = matlab.buildtool.tasks.TestTask();
plan.DefaultTasks = "test";
end
```

Then all three of our example steps (`run-tests`, `run-build`, and `run-command`) are demonstrating what it takes to produce the same behavior.

Reply:
That makes sense. I agree, this is not a buildtool example repo, it's a CI example repo so the simpler the better.

Reply:
+1. 

What do you think of specifying the "tests" argument?
`plan("test") = matlab.buildtool.tasks.TestTask("tests")`

It is not required but it seems nice to have a bit of parallel:
- addpath to "code"
- run tests in "tests". 

makes it slightly more reader-friendly. 

Plus, it seems a tad bit faster to create a suite starting from a directory where there are tests.